metrics/mixin/alerts/perf.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
```

Comment:
Run `make gen-latest-prometheus-rules-yamls` from project root to update the generated rule files.

Reply:
@manishym This is still not fixed.

Reply:
ack